plugins/woocommerce/src/Internal/Admin/Suggestions/Incentives/Incentive.php

```diff
@@ -238,12 +238,21 @@ public function get_dismissals( string $id ): array {
 			return array();
 		}
 
-		$dismissals = array_filter(
-			$dismissed_incentives,
-			fn( $dismissed_incentive ) => $id === $dismissed_incentive['id']
+		$dismissals = array_values(
+			array_filter(
+				$dismissed_incentives,
+				fn( $dismissed_incentive ) => $id === $dismissed_incentive['id']
+			)
 		);
 
-		return array_column( $dismissals, 'context' );
+		// Map to return an array of associative arrays containing both 'context' and 'timestamp'.
```

Comment:
No need for this comment as it is quite evident we want the two columns. Maybe we should have a utility for [this](https://www.php.net/manual/en/function.array-column.php#128387)

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/56396/commits/63106def6e03e33c0d2a461b8bf3b6c9b212bc4a